scripts/earthly-local

```diff
@@ -10,8 +10,7 @@ function start_minio() {
     # Already started
     return
   fi
-  mkdir -p ~/.minio/data
-  docker run -d -p 12000:9000 -p 12001:12001 -v ~/.minio/data:/data \
+  docker run -d -p 12000:9000 -p 12001:12001 -v minio-data:/data \
```

Comment:
Note: this is so I could easily clear the data accumulated in MinIO. With it in the home directory I had to `sudo rm -rf` because the objects are written by root inside the container 🥹

Reply:
makes sense

Reply:
It should create a volume under `/var/lib/docker/volumes`, not in the current dir